go.mod

```diff
@@ -1,10 +1,16 @@
 module github.com/goware/cachestore
 
-go 1.16
+go 1.18
 
 require (
 	github.com/gomodule/redigo v1.8.8
 	github.com/hashicorp/golang-lru v0.5.4
 	github.com/pkg/errors v0.9.1
 	github.com/stretchr/testify v1.7.1
 )
+
+require (
+	github.com/davecgh/go-spew v1.1.0 // indirect
```

Comment:
can you find in the source where we have go-spew use..? we should leave it..

Reply:
go-spew is coming from "github.com/stretchr/testify/assert"
All 3 indirect are coming from "github.com/stretchr/testify/assert"